app/monitor_terminal.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import json
+import os
+import time
+
+import spade
+from spade.agent import Agent
+from spade.behaviour import PeriodicBehaviour
+
+class MonitorAgent(Agent):
+    '''periodacially sends query messages to all sensors'''
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+    
+    async def setup(self):
+        print(f"Monitor agent with id: {self.jid} initialized")
+        b = self.SendQuery(period=5)
+        self.add_behaviour(b)
+    
+    class SendQuery(PeriodicBehaviour):
+        async def run(self):
+            print(f"Monitor agent with id: {self.agent.jid} querying sensors")
+            msg = spade.message.Message(to="sensor@localhost")
```

Comment:
Only sends message to one sensor, should probably be a loop sending messages to a list of sensors.

Reply:
Nope. This line sends to every sensor